packages/emails/src/components/LocationInfo.tsx

```diff
@@ -41,7 +41,7 @@ export function LocationInfo(props: { calEvent: CalendarEvent; t: TFunction }) {
           </a>
         }
         extraInfo={
-          meetingUrl && (
+          meetingUrl && !props.calEvent.requiresConfirmation ? (
```

Comment:
This is also true for the email when the event gets confirmed. This email should include the meeting URL